release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -116,6 +116,12 @@ For this release, Red Hat only supports using MetalLB in layer 2 mode.
 
 For more information, see xref:../networking/metallb/about-metallb.adoc#about-metallb[About MetalLB and the MetalLB Operator].
 
+[id="ocp-4-9-networking-CNI-VRF-plug-in"]
+==== CNI VRF plug-in
+The CNI VRF plug-in was previously introduced as a Technology Preview feature in {product-title} 4.8, and is now generally available in {product-title} {product-version}.
```

Comment:
Just one thing I noticed regarding the version number. In this sentence is says the Technology Preview feature was introduced in 4.8, but in Table 2. Technology Preview tracker TP is also included in the OCP 4.7 column. So either this sentence needs to be changed to 4.7, or in Table 2. Technology Preview tracker TP needs to be removed from the OCP 4.7 column.

Reply:
+1. The VRF code support was added in OCP 4.7.
https://github.com/openshift/containernetworking-plugins/tree/release-4.7/plugins/meta/vrf
https://issues.redhat.com/browse/CNF-846